dnf-behave-tests/features/microdnf/protected-running-kernel1.feature

```diff
@@ -0,0 +1,21 @@
+@no_installroot
+Feature: Protected running kernel
+
+
+@bz1698145
+Scenario: Running kernel is protected
+  Given I use repository "repoquery-unsatisfied"
+   When I execute microdnf with args "install dnf-ci-kernel"
+   Then the exit code is 0
+    And microdnf transaction is
+        | Action        | Package                            |
+        | install       | dnf-ci-kernel-0:1.0-1.x86_64       |
+        | install       | dnf-ci-systemd-0:1.0-1.x86_64      |
```

Comment:
I think instead of the three steps above you can just use:
```gherkin
Given I successfully execute microdnf with args "install dnf-ci-kernel"
```
(if on accident nothing would get installed and `microdnf` exited with 0, the tests would surely fail later on?)

You can also now put multiple microdnf tests into a single .feature file, the runner now launches them in separate containers.

And if you do that, you can put the shared `Given` steps into `Background` to save on repetition :slightly_smiling_face:

Reply:
Done.